client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -61,38 +61,38 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
             .finally(() => setIsLoading(false));
     };
 
-    const getRulerApiDataFromServer = async (ids : any []) => {
+    const getRulerApiDataFromServer = async (ids: any[]) => {
         const RulerCheckColorRequestParameters = {
-            "RulerCheckColorRequest":{     
-                "MOHHeader":{       
-                    "ActivationID":"1",     
-                    "CustID":"23",
-                    "AppID":"123",
-                    "SiteID":"2",       
-                    "InterfaceID":"Ruler"
+            "RulerCheckColorRequest": {
+                "MOHHeader": {
+                    "ActivationID": "1",
+                    "CustID": "23",
+                    "AppID": "123",
+                    "SiteID": "2",
+                    "InterfaceID": "Ruler"
                 },
-                "Ids":ids
+                "Ids": ids
             }
         }
-            
+
         const rulerLogger = logger.setup('client ruler logger setup');
         rulerLogger.info(`launching server request with parameter: ${JSON.stringify(RulerCheckColorRequestParameters)}`, Severity.LOW);
         setIsLoading(true);
         return await axios.post('/ruler/rulerapi', RulerCheckColorRequestParameters)
-        .then((response: any) => {
```

Comment:
can you please discard these changes it will cost conflicts and i prefer to fix these spaces after everything will work...